addons/account/models/account_move_send.py

```diff
@@ -691,29 +692,33 @@ def _check_sending_data(self, moves, **custom_settings):
         ) if 'sending_methods' in custom_settings else True
 
     @api.model
-    def _generate_and_send_invoices(self, moves, from_cron=False, allow_raising=True, allow_fallback_pdf=False, **custom_settings):
+    def _generate_and_send_invoices(self, moves, *, allow_raising=True, allow_fallback_pdf=False, notification=None, **custom_settings):
```

Comment:
Putting `*` is not making the code cleaner imo. Why removing completely `from_cron` ?
Have you planned to manage it with `custom_settings` but you forgot about it?

Reply:
`*` forces the arguments to be used as kw-args, for boolean flags, positional arguments make little sense and are hard to read.
Whether you come from a cron can be deduced by reading context values.